internal/cmd/server.go

```diff
@@ -111,3 +123,12 @@ func validateServerCommandArguments(cmd *cobra.Command, args []string) error {
 
 	return nil
 }
+
+func processLogs() {
+
+	for {
+		if !logger.ProcessMsg(logger.SCRIPTS) {
```

Comment:
Why? it's so obscure. Let's go with these, I'd like to refactor in the future.

Reply:
don't understand what is obscure?